### ISSUE TITLE
Payments: Manage Subscribers Options are extremely difficult to see in Calypso

### ISSUE THREAD

**Issue by scosgro** [Automattic/wp-calypso#46253]
#### Steps to reproduce

1. Have someone subscribed to your site (through Payments)
2. Go from your WordPress.com dashboard to Tools - Earn - Collect Payments, and view your subscribers under Customers and Subscribers. 
3. To the right of each customer/subscriber should be a ··· icon to manage their subscription, or view it in Stripe.
4. The ··· is there, but it's hardly visible. It blends in to the Calypso background.

#### What I expected
The ··· to be visible.

#### What happened instead
The ··· blends into the background on the page. It's only faintly noticeable. 

#### Browser / OS version
All.

#### Screenshot / Video
Screenshots sanitized of user information

![Screenshot 2020-10-07 174716](https://user-images.githubusercontent.com/30731356/95402759-53a78880-08c5-11eb-874a-e1101126d85d.png)

And

![Screenshot 2020-10-07 174744](https://user-images.githubusercontent.com/30731356/95402767-586c3c80-08c5-11eb-9453-84396da9c5b7.png)

#### Context / Source
#user-report

**Comment by scosgro** [comment]
The options for those items are still fairly difficult to see:
![image](https://user-images.githubusercontent.com/30731356/114225097-c7832600-9926-11eb-9ca7-264a65c046e1.png)
and
![image](https://user-images.githubusercontent.com/30731356/114225127-cfdb6100-9926-11eb-94a5-beb4ebd5b9c5.png)

**Comment by scosgro** [comment]
Another report in 28983333-hc

**Comment by scosgro** [comment]
Another report in 26308629-hc

**Comment by inaikem** [comment]
Closing in favour of #43227
